### PR TITLE
Create Belfast file

### PR DESCRIPTION
**Pull request by darrenaitcheson**
This is a test.

### CODE REVIEW

File: darrenaitcheson/Belfast.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Belfast
+
+### Restaurants :pizza:
+
+- Simply Fish & Chips
+- KFC
+- Il Pirata
```
Comment (khallai) [inline darrenaitcheson/Belfast.md:7]: @darrenaitcheson What type of restaurant is this?

File: darrenaitcheson/Belfast.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Belfast
+
+### Restaurants :pizza:
+
+- Simply Fish & Chips
+- KFC
```
Comment (tlcoder) [inline darrenaitcheson/Belfast.md:6]: Yum! :chicken:

File: darrenaitcheson/Belfast.md (hunk 1 above)
Comment (darrenaitcheson) [inline darrenaitcheson/Belfast.md:7]: Italian :smile:

**Comment by jokan1** [comment]
Interesting

**Pull request merged**